arangod/Aql/ConstFetcher.cpp

```diff
@@ -29,7 +29,6 @@
 #include "Aql/SkipResult.h"
 #include "Basics/Exceptions.h"
 #include "Basics/voc-errors.h"
-#include "Containers/SmallVector.h"
```

Comment:
Shouldn't this include be kept now, given that SmallVector is used in this file but is not included in the header anymore?